### DISCUSSION TITLE
Requiring people to sign up with vercel to do the next.js tutorial is super corporate and comes off weirdly aggressive

### DISCUSSION THREAD

**Discussion by iiian** [vercel/community#5640] [Help] [answered]
### Summary

https://nextjs.org/learn/dashboard-app/fetching-data

Look, I get that you guys corporate, so like whatever... but shoe-horning product placement for `@vercel/postgres` into a tutorial like this--without providing any sort of an open source alternative--just kind of puts a bad taste in my mouth as a developer. It feels very aggressive: *"give us your data, now. we want to watch you, piggy."* 

Like, I wasn't going to say anything but `@vercel/postgres` doesn't seem to work without downloading the `vercel` cli to get the `vercel env pull .env.development.local`. That requires me to give `vercel link`, which then wants to link to my **projects**? Like, uhhhhh, what? How invasive do you guys really need to get for a **tutorial?**

Anyways, gross gross gross. I know you won't do anything to change your behavior because it's bad for business or some similar *bs* justification. C'est la vie.

### Steps to Reproduce

👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎👎

**Comment by leerob** [comment, marked as answer]
You can use any Postgres service if you don't want to use the free service of Neon integrated through Vercel.

Here's a similar client: https://github.com/porsager/postgres
